api/src/main/java/org/openmrs/module/patientflags/db/FlagDAO.java

```diff
@@ -228,4 +228,12 @@ public interface FlagDAO {
 	 * @throws DAOException
 	 */
 	public void purgeDisplayPoint(Integer displayPointId) throws DAOException;
+
+	/**
+	 * Checks if given priority's name is duplicated
+	 *
+	 * @param priority object to verify uniqueness of priority name
+	 * @return true or false depending on priority's name is duplicated or not
+	 */
+	boolean isPriorityNameDuplicated(Priority priority);

```

Comment:
Refer to my comment [above](https://github.com/openmrs/openmrs-module-patientflags/pull/36/files#r865871231)